docs/en_us/platform_api/source/links.rst

```diff
@@ -1 +1,7 @@
-.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
\ No newline at end of file
+.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
+
+.. _edX platform documentation: http://docs.edx.org/
```

Comment:
erm, I meant, can we just call it "docs.edx.org" rather than naming it the "edX platform documentation"? I think that sometimes people confuse "edX platform" with the edx-platform repo, which isn't really so...